proj/src/main/webapp/script.js

```diff
@@ -366,4 +373,54 @@ function changeChatOnClick(domElement, tag, chatId) {
         dbRefObject = getDbRef(tag, chatId);
         initRef();
     });
+}
+
+function populateProfileSidebar() {
+    firebase.auth().onAuthStateChanged(function(user) {
+        if (user) {
+        const userRef = firebase.database().ref('/users/'+user.uid)
+        userRef.once('value', snap => {
+            const userObj = {};
+            userObj.photo = user.photoUrl;
```

Comment:
I'm not entirely sure how images are stored in firebase. Are they stored as a URL or as some sort of blob or something? I guess it depends on if you make users upload images or specify a URL to a photo already existing on the web to use

Reply:
yep, they're stored as a URL and saved to the firebase auth database